### PR TITLE
added dominoes exercise

### PR DESCRIPTION
**Pull request by massivefermion**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -222,14 +222,22 @@
...
           "math"
         ]
+      },
+      {
+        "slug": "dominoes",
+        "name": "Dominoes",
+        "uuid": "b59b630e-8648-4498-b8d8-fcfaf261ece3",
+        "difficulty": 7,
+        "practices": [],
+        "prerequisites": []
```
Comment (lpil) [inline config.json:240]: Could you fill these in please? Thanks!

**Review by lpil — COMMENTED** [review]
Fab, thank you!

There's a lot of reversing and accessing the ends of list here. These are very slow operations that should best be avoided. I think the queue data structure would potentially be a better fit here as it can be quickly accessed from both ends.

Can we also remove the use of `assert`? Best to avoid that as much as possible.

**Comment by massivefermion** [comment]
I changed things as you requested and everything seems to be fine. But for some unknown reason the checks are failing!

**Review by lpil — COMMENTED** [review]
Yup that's right, the test runner is not on the latest version of Gleam yet. Probably easiest to not use `use` just yet for this.

**Comment by massivefermion** [comment]
ok, done!

**Review by lpil — APPROVED** [review]
You star, thank you!

**Pull request merged**